src/SpotifySession.ts

```diff
@@ -10,6 +10,15 @@ export default interface SpotifySession {
      */
     accessToken: string;
 
+
+    /**
+     * (Android) The access code returned when using the CODE auth type
+     *
+     * @type {string}
+     * @memberof SpotifySession
+     */
+    code?: string;
```

Comment:
@reteps would you have a quick gander at the changes?  Also, I've updated the example app.. but I'm currently using the CODE / TOKEN interchangeably.  What is the expected behavior of the CODE type?  

You were mentioning being able to skip the auth screen.  When I use the code type it still does bring up the spinner for a moment.  It seems that either the `accessToken` or the `code` can be used to connect to the spotify remote.  So I think I can probably remove this from the API and just return either the code or the token in the `accessToken` property.

Anyways, I can't find any docs on the CODE option so I'm not sure what the expectation is.

Reply:
This is a bad coding practice. `CODE` is an auth flow for different needs (e.g. when you need to refresh tokens on your server), and inserting both `code` and `accessToken` in one `accessToken` property made me think this feature is actually not working / not supported. After a couple of hours of research, I luckily found this PR. The only way I became sure the `CODE` flow is working is because I've compared the length of the `accessToken` field.

Grateful for your work on the library, but please don't use one field for two completely different things.